security-analysis/security-analysis-api/src/main/java/com/powsybl/security/PostContingencyResult.java

```diff
@@ -35,11 +55,28 @@ public PostContingencyResult(Contingency contingency, boolean computationOk, Lis
         this(contingency, new LimitViolationsResult(computationOk, limitViolations, actionsTaken));
     }
 
+    public PostContingencyResult(Contingency contingency, boolean computationOk, List<LimitViolation> limitViolations,
+                                 List<String> actionsTaken, Map<String, BranchResult> branchResults, Map<String, BusResults> busResults, Map<String, ThreeWindingsTransformerResult> threeWindingsTransformerResults) {
+        this(contingency, new LimitViolationsResult(computationOk, limitViolations, actionsTaken), branchResults, busResults, threeWindingsTransformerResults);
+    }
+
     public Contingency getContingency() {
         return contingency;
     }
 
     public LimitViolationsResult getLimitViolationsResult() {
         return limitViolationsResult;
     }
+
+    public List<BranchResult> getBranchResultsAsList() {
+        return new ArrayList<>(branchResults.values());
```

Comment:
A detail but for all those getters I would advise to use `List.copyOf`, to return an immutable copy instead of a mutable one.